framework/doc/content/source/ics/IntegralPreservingFunctionIC.md

```diff
@@ -0,0 +1,42 @@
+# IntegralPreservingFunctionIC
+
+!syntax description /ICs/IntegralPreservingFunctionIC
+
+## Description
+
+Sets an initial condition while preserving an integral.
+The [!param](/ICs/IntegralPreservingFunctionIC/function) is used to specify the overall "form" of the initial condition,
+such that the applied initial condition is
+
+\begin{equation}
+\label{eq:ic1}
+u=q_0f
+\end{equation}
+
+where $u$ is the variable, $f$ is the function, and $q_0$ is a scaling
+factor used to preserve a total [!param](/ICs/IntegralPreservingFunctionIC/magnitude) upon volume integration:
+
+\begin{equation}
+\label{eq:ic2}
+q_0=\frac{Q}{\int_\Omega fd\Omega}
+\end{equation}
+
+where $Q$ is the total magnitude and $\Omega$ is the domain of integration.
```

Comment:
is Q good english for an integral?

Magnitude feels like amplitude, like a coefficient on the value more than the integral

Reply:
Q just represents the integral of the scaled function (I would say that `q_0` in this notation actually represents the amplitude). Magnitude to me is different from amplitude; to me, magnitude is like the size of a vector -- but here, instead of a norm as a way to get a scalar value from a many-valued field, I'm referring to magnitude as the integral of that many-valued field.

Do you want me to change anything here? I think the documentation is pretty clear

Reply:
yeah I think it s great. Was just asking